crates/hotshot-signature-key/src/bn254/bn254_pub.rs

```diff
@@ -134,6 +134,12 @@ impl SignatureKey for BN254Pub {
         }
     }
 
+    fn get_public_key(entry: &Self::StakeTableEntry) -> Self {
```

Comment:
This function is part of `impl SignatureKey for BN254Pub`, so I guess the `SignatureKey` trait already has documentation, and lints won't require this function here to also have a description. Otherwise, I agree we should add more descriptions here!

Reply:
Yes! the description is [here](https://github.com/EspressoSystems/HotShot/pull/1738/files#diff-397259bb8b14663db4645bf156fc7abebca2120237f8712adad9a9c14db5a5a2R104) in the `SignatureKey` trait.